### DISCUSSION TITLE
Projects

### DISCUSSION THREAD

**Discussion by saturninosegurola** [federicoVisi/SoundS-gesture-sound-interaction#4] [Ideas]
Folder to upload projects updates

**Comment by federicoVisi** [comment]
Hi Mariano, check the email I sent you earlier, I added you as collaborator, please upload your files here: https://github.com/federicoVisi/SoundS-gesture-sound-interaction/tree/main/Lab/Barry%2BMariano

**Comment by maggsss** [comment]
https://drive.google.com/file/d/1d9iR462iiXUdggK4LSgQeW5gi2A3pLng/view?usp=sharing

**Comment by maggsss** [comment]
powern point:
https://docs.google.com/presentation/d/1WuXYXB7X0FcXLX9GlAqcT7xMp6hu7lCm/edit?usp=sharing&ouid=117669625693558861679&rtpof=true&sd=true
